apps/skilavottord/ws/src/app/modules/accessControl/accessControl.input.ts

```diff
@@ -0,0 +1,32 @@
+import { Field, InputType } from '@nestjs/graphql'
+
+@InputType()
+export class CreateAccessControlInput {
+  @Field()
+  nationalId!: string
+
+  @Field()
+  name!: string
+
+  // TODO: use the Role enum and create GraphQL Enum for it as well
+  @Field()
+  role!: string
+
+  // TODO: get from samgongustofa
+  @Field()
+  partnerId!: string
+}
+
+@InputType()
+export class UpdateAccessControlInput {
+  @Field()
```

Comment:
Just figured out we need the nationalId here as well, since otherwise we don't know what field we're updating.

Reply:
Should I leave both inputs and make them the same or just add AccessControlInput for both?